### ISSUE TITLE
Ingest Health Dashboards

### ISSUE THREAD

**Issue by P1llus** [elastic/integrations#5245]
This issue is to track ongoing work to create and use some general dashboards that will be bundled with the elastic-agent integration together with the existing Agent Metrics dashboard.

Currently there will be a few different dashboards:

1. Overview Dashboard that shows a more general overview of ingest health and statistics.
2. Agent Dashboard that shows statistics grouped by agent, like ingest errors, integrations installed, different versions used in the stack etc.
3. Integration Dashboard, that shows statistics grouped by integrations.

**Comment by jamiehynds** [comment]
@dhru42 is leading the Data Quality dashboard initiative and we should aim to have some cohesiveness between both efforts. 

@dhru42 how are you planning on shipping the Data Quality dashboards, e.g. via stack monitoring, an integration package or potentially bundling with elastic-agent? Are you targeting a specific release? 

pinging @nimarezainia too for visibility as he and I discussed the ingest health dashboards recently too.

**Comment by P1llus** [comment]
The current scope of the plan, working with @nimarezainia and @joshdover is that the dashboards, once finalized, would be bundled with the elastic-agent package @jamiehynds

**Comment by dhru42** [comment]
> @dhru42 is leading the Data Quality dashboard initiative and we should aim to have some cohesiveness between both efforts.
> 
> @dhru42 how are you planning on shipping the Data Quality dashboards, e.g. via stack monitoring, an integration package or potentially bundling with elastic-agent? Are you targeting a specific release?
> 
> pinging @nimarezainia too for visibility as he and I discussed the ingest health dashboards recently too.

@jamiehynds the current dashboard will be released in the security solution under the `BETA` flag, [link](https://github.com/elastic/kibana/pull/150063) to the PR.


cc: @andrew-goldstein @ARWNightingale @MikePaquette

**Comment by P1llus** [comment]
> @P1llus can we add the 'Security' tag to the ingest health dashboards to ensure they appear under the 'Custom' section of the security dashboards page? 
> 
> <img width="1984" alt="Screenshot 2023-02-13 at 16 52 21" src="https://user-images.githubusercontent.com/62879768/218521353-a0e8d816-92df-4be4-907d-bc67e5906bff.png">
> 

Currently its not possible unfortunately, this is because each package will have a unique tag called "Security", so for each package installed, a duplicated tag is created.
But we can try to look for a workaround.
Its also good to remember that the dashboard will already be linked in the fleet UI at some point in the near future.

**Comment by jamiehynds** [comment]
Thanks @P1llus. Although it will appear in the Fleet UI, I think it's important to have it included somewhere in the Security app for visbility/discoverability too. Dashboards is the logical place. If there's any workarounds we can think of, it makes sense to spend time on it (depending on level of effort of course). 

I think we ran into a similar issue when we shipped the Threat Intel Utilities package and there was an issue for it somewhere, presumable Kibana or Fleet.
